actions/stack/release-notes/action.yml

```diff
@@ -21,22 +21,22 @@ inputs:
     description: 'CVE scan report path of run image in markdown format'
     required: false
   build_packages_added:
-    description: 'JSON array of packages added "{ name: name, version: 1.2.3 }"'
+    description: 'Path to build packages added file'
```

Comment:
I think these should be `required: true` as it will fail faster and with a better error message vs letting the empty value propagate to the go code and then panicing.

Thoughts?

Reply:
Oh, sorry, i was looking at the wrong file. Ignore me 😄